### DISCUSSION TITLE
EasyBackup git error on "View status checks" Page

### DISCUSSION THREAD

**Discussion by Rudiberto** [kimai/kimai#4732] [Plugins]
EasyBackup seems to work, but showing this message:
`git 	fatal: detected dubious ownership in repository at '/var/www/kimai/var/plugins/EasyBackupBundle' To add an exception for this directory, call: git config --global --add safe.directory /var/www/kimai/var/plugins/EasyBackupBundle `
I've tried the recommendation written there, but it doesn't solve the issue. So what ownership is working for the directory? What is the reason for the error message, since I followed the instructions step by step?

**Comment by kevinpapst** [comment]
Is that the Doctor screen?

**Comment by Rudiberto** [comment]
Nein. Doctor is installed, but it's not his screen
![grafik](https://github.com/kimai/kimai/assets/96433043/fa520912-f96f-48e0-b303-81d089309465)

**Reply by kevinpapst** [reply]
Ah okay, can you then please post in the repo of the plugin, link is in the store page.

**Reply by Rudiberto** [reply]
![grafik](https://github.com/kimai/kimai/assets/96433043/5a50552f-ebd0-45f9-b8a2-8683ab95a528)
so sollte es aussehen. Looks like a Hash.

**Comment by Rudiberto** [comment]
<offtopic>
by the way:
Moin, super Programm!
</offtopic>
